webapp/tests/setup.tsx

```diff
@@ -1,7 +1,11 @@
-import 'mattermost-webapp/tests/setup';
 import '@babel/polyfill';
 import {jest} from '@jest/globals';
 
+import Enzyme from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
```

Comment:
CI is not happy

```
Error: tests/setup.tsx(5,21): error TS7016: Could not find a declaration file for module 'enzyme-adapter-react-16'. '/home/runner/work/matterpoll/matterpoll/webapp/node_modules/enzyme-adapter-react-16/build/index.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/enzyme-adapter-react-16` if it exists or add a new declaration (.d.ts) file containing `declare module 'enzyme-adapter-react-16';`
make: *** [Makefile:49: check-style] Error 2
Error: Process completed with exit code 2.
```
https://github.com/matterpoll/matterpoll/actions/runs/8589443726/job/23559354710?pr=251

Reply:
I didn't really touch JS other than to test it on Mac. I needed to pull in an update at some point to make it build. Could be related to that. Any advise on how to fix this? I think I'll just take revert to the current master for all files in webapp to be sure.